reactioncommerce/api-service-account
# feat: create this plugin (and add semantic-release to CI)

This service replaces the internal `accounts` package that is currently in Reaction. It has not been tested by anyone aside myself yet, so although the changes in this particular PR only enable semantic release, please test the full functionality of the service before approving.

Once this PR is merged and this plugin is published to `npm`, there will be a related PR in `reaction` to swap out this plugin for the existing version.

## Testing

- `npm link` this package into the Reaction API
- remove the existing `src/core-services/account` folder to delete the current version of this plugin
- update `registerPlugins.js` inside of Reaction API to use the npm version of this package
- Start Reaction and create an initial user, this will prove the plugin is working
- Perform other account related queries / mutations to confirm they work
